src/world/common/DeadUnkNpcAIMainFunc5.inc.c

```diff
@@ -0,0 +1,143 @@
+
+    territory.unk_00 = 0;
+    territory.shape = enemy->territory->wander.detectShape;
+    territory.pointX = enemy->territory->wander.detect.x;
+    territory.pointZ = enemy->territory->wander.detect.z;
+    territory.sizeX = enemy->territory->wander.detectSizeX;
+    territory.sizeZ = enemy->territory->wander.detectSizeZ;
+    territory.unk_18 = 100.0f;
+    territory.unk_1C = 0;
+
+    enemy->unk_108.x = npc->pos.x;
+    enemy->unk_108.y = npc->pos.y;
+    enemy->unk_108.z = npc->pos.z;
+    enemy->unk_114 = 0.01f;
+    enemy->unk_118 = 0.01f;
+
+    if (isInitialCall) {
+        enemy->varTable[6] = npc->collisionHeight;
+        enemy->varTable[8] = 0;
+        enemy->unk_B5 = 0;
+        enemy->unk_B0 |= 8;
+    }
+
+    if (isInitialCall || (enemy->unk_B0 & 4)) {
+        script->functionTemp[0] = 0;
+        npc->duration = 0;
+        enemy->unk_07 = 0;
+        npc->currentAnim.w = enemy->animList[0];
+        npc->flags &= ~NPC_FLAG_NO_Y_MOVEMENT;
+        npc->collisionHeight = enemy->varTable[6];
+        enemy->varTable[9] = 0;
+
+        if (!enemy->territory->wander.isFlying) {
+            npc->flags |= NPC_FLAG_GRAVITY;
+            npc->flags &= ~NPC_FLAG_ENABLE_HIT_SCRIPT;
+        } else {
+            npc->flags &= ~NPC_FLAG_GRAVITY;
+            npc->flags |= NPC_FLAG_ENABLE_HIT_SCRIPT;
+        }
+
+        if (enemy->unk_B0 & 4) {
+            s32 emoteTemp;
+
+            script->functionTemp[0] = 99;
+            script->functionTemp[1] = 0;
+            fx_emote(2, npc, 0.0f, npc->collisionHeight, 1.0f, 2.0f, -20.0f, 0x28, &emoteTemp);
+            enemy->unk_B0 &= ~4;
+        } else if (enemy->flags & ENEMY_FLAGS_40000000) {
+            script->functionTemp[0] = 12;
+            enemy->flags &= ~ENEMY_FLAGS_40000000;
+        }
+    }
+
+    if (enemy->varTable[9] > 0) {
+        enemy->varTable[9]--;
+        if (enemy->varTable[9] == 0) {
+            if (npc->currentAnim.w == 0x55002E || npc->currentAnim.w == 0x55002F) {
+                npc->currentAnim.w = 0x55000C;
+            }
+        } else {
+            return ApiStatus_BLOCK;
+        }
+    }
+
+    switch (script->functionTemp[0]) {
+        case 0:
+            func_800495A0(script, aiSettings, territoryPtr);
+            npc->collisionHeight = enemy->varTable[6];
+        case 1:
+            func_800496B8(script, aiSettings, territoryPtr);
+            break;
+        case 2:
+            base_UnkNpcAIFunc1(script, aiSettings, territoryPtr);
+            if (enemy->varTable[7] == 6) {
+                if (rand_int(100) < 33) {
+                    if (enemy->varTable[8] != 0) {
+                        enemy->varTable[8] = 0;
+                        enemy->unk_B5 = 0;
+                        npc->currentAnim.w = 0x55002F;
+                    } else {
+                        enemy->varTable[8] = 1;
+                        enemy->unk_B5 = 1;
+                        npc->currentAnim.w = 0x55002E;
```

Comment:
These anims should have defined values? Don't know what npc offhand

Reply:
see ver/us/build/include/sprite/npc/thrown_bone.h